recipes/cosmosis-build-standard-library/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cosmosis-build-standard-library" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d818aa4f0bebf3d5507014a7042caf1f39636215a348322129647e51fa874055
+
+build:
+  number: 0
+  skip: True # [(py < 36) or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cosmosis
+    # For compiling standard librarymodules
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - {{ mpi }}
+    - bzip2
+    - cfitsio
+    - gsl
+    - fftw
+    - libblas
+    - liblapack
+    - camb
+    # for running python standard library modules
+    - fitsio
+    - astropy
+    - cython
+    - scikit-learn
+
+
+test:
+  commands:
+    - cosmosis-build-standard-library && cd cosmosis-standard-library && cosmosis demos/demo1.ini && cosmosis-postprocess demos/demo1.ini
+    - test -f matter_power.png
+
+about:
+  home: https://github.com/joezuntz/cosmosis
+  license: BSD-2-Clause
```

Comment:
If the pypi package vendors code from other authors, even if that code is not compiled, you will need to put the licenses here. Even if the pypi package has the licenses, we need them in a place conda will grab them and package them. If you store them in some spot upon install, that will work. Files that are simply in the pypi sdist may or may not make it.

Reply:
cosmosis-build-standard-library doesn't include any vendored code yet (or almost any code at all really), so nothing to do here. But I've done that below for the other package.

Licenses might be a significant blocker for packaging the CSL.  The Planck lib (among others) has no license included, just an "All rights reserved" copyright statement in the files.

Reply:
Yep we can't ship unlicensed code.